CMakeLists.txt

```diff
@@ -294,7 +297,7 @@ if(GENERATE_DOXYGEN_DOC)
             ${HELICS_CURRENT_BINARY_DIR}/Doxyfile @ONLY
         )
         add_custom_target(
-            doc ${DOXYGEN_EXECUTABLE} ${HELICS_CURRENT_BINARY_DIR}/Doxyfile
+            helics_doc ${DOXYGEN_EXECUTABLE} ${HELICS_CURRENT_BINARY_DIR}/Doxyfile
```

Comment:
Maybe this target could be renamed to `helics_doxygen` (or something similar) to make it clear that it is just running doxygen rather than building all of the docs.

Reply:
I think I renamed it since it was conflicting with something else.   I am working on a more substantial doc update PR,  I will rename it in that PR to helics_doxygen, there were some other issues with this project that I am also resolving in that PR.